EloquaClient.Tests/Unit/EloquaContextTests.cs

```diff
@@ -139,5 +141,98 @@ public async Task DbSetUpdateCustomCampaignObjectAsyncFailTest()
             Assert.IsTrue(result.HasError);
         }
         #endregion
+
+        #region SearchCustomCampaignObjectAsync
+        [TestMethod]
+        public async Task SearchCustomCampaignObjectAsyncPassTest()
+        {
+            //Arrange
+            const string customObjectJson = @"{
+                'elements': [
+                {
+                    'type': 'CustomObjectData',
+                    'id': '10',
+                    'createdAt': '1538070228',
+                    'depth': 'minimal',
+                    'name': 'test@test.com',
+                    'updatedAt': '1538162787',
+                    'contactId': '11',
+                    'fieldValues': [
+                    {
+                        'type': 'FieldValue',
+                        'id': '20',
+                        'value': 'test@test.com'
+                    },
+                    {
+                        'type': 'FieldValue',
+                        'id': '21',
+                        'value': '0'
+                    },
+                    {
+                        'type': 'FieldValue',
+                        'id': '22',
+                        'value': '0'
+                    }
+                    ],
+                    'isMapped': 'Yes',
+                    'uniqueCode': 'test@test.com'
+                }
+                ],
+                'page': 1,
+                'pageSize': 1000,
+                'total': 1
+            }";
+            var mockRestResponse = new Mock<IRestResponse>();
+            mockRestResponse.SetupGet(o => o.ResponseStatus).Returns(ResponseStatus.Completed);
+            mockRestResponse.SetupGet(o => o.StatusCode).Returns(HttpStatusCode.OK);
+            mockRestResponse.SetupGet(o => o.Content).Returns(customObjectJson);
+
+            var mockRestClient = new Mock<IRestClient>();
+
+            //Act
+            var eloquaContext = new EloquaContext(mockRestClient.Object);
+
+
+            mockRestClient.Setup(o => o.ExecuteTaskAsync(It.IsAny<IRestRequest>())).ReturnsAsync(mockRestResponse.Object);
+
+
+            //Act
+            var result = await eloquaContext.SearchCustomCampaignObjectsAsync("test@test.com");
+
+            //Assert
+            Assert.IsInstanceOfType(result, typeof(List<CustomObjectData>));
+
+            
+            Assert.AreEqual(10, result[0].Id);
+        }
+        #endregion
+
+        #region SearchCustomCampaignObjectAsync
```

Comment:
rename region to CreateCustomCampaignObjectsAsync ?

Reply:
Should be fixed.  It was a copy error...  :-/